src/admin/paths/admin@reports@add_approvers.yaml

```diff
@@ -0,0 +1,46 @@
+post:
+  tags:
+    - Reports
+  summary: Add Approvers
+  description: |
+    Add Approvers
+  operationId: report_add_approvers
+  requestBody:
+    required: true
+    content:
+      application/json:
+        schema:
+          type: object
+          additionalProperties: False
+          required:
+            - data
+          properties:
+            data:
+              $ref: ../../components/schemas/report.yaml#/add_approvers_in
+  responses:
+    '200':
```

Comment:
remove approver is not bulk
so why the response is simple 200
it should ideally be report object

Reply:
arey sir khudka bhi lagao na, galti ho sakti hai